compat/win32/ntifs.h

```diff
@@ -0,0 +1,139 @@
+#ifndef _NTIFS_
+#define _NTIFS_
+
+/*
+ * Copy necessary structures and definitions out of the Windows DDK
```

Comment:
Necessary for what? ;-)

Maybe mention that this really defines and declares the constants and signatures necessary to call `NtQueryDirectoryFile()`?

And I assume that it is okay (license-wise) to just copy those? Did you copy them from `mingw-w64`'s headers? If so, that copying should be covered by the license.